netbout-web/src/main/java/com/netbout/rest/bout/TkInvite.java

```diff
@@ -84,11 +114,89 @@ public Response act(final Request req) throws IOException {
                 ),
                 Level.INFO
             )
         );
     }
 
+    /**
+     * Invite a user to participate on NetBout by email.
+     * @param invite Email.
+     * @return Alias.
+     * @throws IOException If fails.
+     * @todo #602:30min/DEV Invited user should receive an email message with
+     *  a text like this 'You are invited into the Netbout click on the link
+     *  to register' and the link should be
+     *  `http://www.netbout.com/b/<bout_number>?invite=<invite-key>`
+     *  where `invite-key` is the encrypted urn. We can use `CcAES` from Takes
+     *  project.
+     */
+    public String inviteByEmail(@NotNull(message = "Invite can't be NULL")
+        final String invite) throws IOException {
+        try {
+            // @checkstyle MultipleStringLiteralsCheck (1 line)
+            final String alias = invite.replace("@", "-").replace(".", "-");
+            final String urn = String.format(
+                "urn:email:%s",
+                this.calcSha(alias)
+            );
+            this.base.user(URN.create(urn)).aliases().add(alias);
+            new RqAlias(this.base, new RqWithAuth(urn)).alias().email(invite);
+            return alias;
+        } catch (final NoSuchAlgorithmException ex) {
+            throw new IllegalStateException(
+                String.format(
+                    "It was not possible to invite \"%s\", try again",
+                    invite
+                ),
+                ex
+            );
+        }
+    }
+
+    /**
+     * Generate an insecure hash of the given message.
+     * @param message Content to be hashed.
+     * @return Hash
+     * @throws NoSuchAlgorithmException If fails to get SHA1 algorithm.
+     * @throws IOException If fails to hash.
+     * @todo #602:30min/DEV All calcSha1 methods should be moved to a class
+     *  that is responsible for hashing insecurely. And existent calls should
+     *  be changed to use that new class. Remember that there is a unit test
+     *  that should be moved to a new class.
+     * @checkstyle ThrowsCountCheck (5 lines)
+     */
+    public String calcSha(@NotNull(message = "Message can't be NULL")
+        final String message) throws IOException, NoSuchAlgorithmException {
+        return this.calcSha(new ByteArrayInputStream(message.getBytes()));
+    }
+
+    /**
+     * Generate an insecure hash of the given message.
+     * @param stream Stream to be hashed.
+     * @return Hash
+     * @throws NoSuchAlgorithmException If fails to get SHA1 algorithm.
+     * @throws IOException If fails to hash.
+     * @checkstyle ThrowsCountCheck (15 lines)
+     */
+    public String calcSha(@NotNull(message = "Input Stream can't be NULL")
+        final InputStream stream) throws IOException, NoSuchAlgorithmException {
+        final byte[] buffer = new byte[BUFFER_SIZE];
+        final MessageDigest digest = MessageDigest.getInstance("SHA1");
+        final DigestInputStream dis = new DigestInputStream(
```

Comment:
@antonini code would be easier to read if you could use `DigestUtils.sha1Hex(...)` from apache commons-codec